src/components/blood-pressure/styles/bp-chart-axes.module.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../styles/mixins.module.scss';
+
+%bp-axis-marker {
+  position: absolute;
+  outline: 0.5px solid black;
+}
+
+%bp-axis-label {
+  position: absolute;
+}
+
+.bp-systolic-axis-marker {
```

Comment:
Could we prefix all the class names with `nhsuk-`? Just to ensure we follow the standard elsewhere and it minimises the chance of clashes with other classes defined by consumers.
Comment is on this file but it will need to be done against all the others too

Reply:
all class name have been prefixed with `nhsuk- `